.travis.yml

```diff
@@ -7,6 +7,7 @@ env:
   - GOARCH=amd64
   - GOARCH=386
 sudo: false
+go_import_path: github.com/nsqio/nsq
```

Comment:
Can you explain why this is needed?

Reply:
@mreiferson without this line - fork build fails, you can see example run here:

https://travis-ci.org/soar/nsq/jobs/294934597

```
package github.com/soar/nsq/apps/nsq_pubsub
    imports github.com/nsqio/nsq/internal/app: use of internal package not allowed
```

And here is Travis documentation for this line:

https://docs.travis-ci.com/user/languages/go/#Go-Import-Path

With this line any fork will work.